doc/sphinx/source/community/detailed_release_procedure.rst

```diff
@@ -0,0 +1,190 @@
+via batch scripts submitted to a schedulers (SLURM). Generate the submission scripts using the `generate.py` Python script;
+you can find a copy of the script either in `/home/b/b382109/Tool_Release_270_Scripts` or
+in the draft Pull Request https://github.com/ESMValGroup/ESMValTool/pull/2883.
+
+You will have to set the name of your environment, your email address (if you want to get email notifications for successful/failed jobs) and the name of the directory you want to store the job outputs. The name of the account is the same (`bk1088`), and the default partition is set to `compute`.
+
+Some recipes need other job requirements, you can add their headers in the `SPECIAL_RECIPES` dictionary. Otherwise the header will be written following the template that is written in the lines below. If you want to exclude recipes, you can do so by uncommenting the exclude lines.
+
+The launch scripts will be saved in the same directory you execute the script from. You can find the ones used for the v2.7.0 release in `/home/b/b382109/submit`
+
+To submit the scripts use the `sbatch` submit scripts (that make use of the SLURM scheduler) produced by `generate.py`,
+and execute them like any other shell script. You can check the status of your BATCH queue by invoking:
+
+.. code-block:: bash
+
+  squeue -u b382109
+
+Also, for computationally-heavy recipes, you can require more memory and/or time, see e.g. edited batch header below
+(note the `compute` partition which is used for such heavy runs):
+
+.. code-block:: bash
+
+  #SBATCH --partition=compute
+  #SBATCH --time=08:00:00
+  #SBATCH --constraint=512G
+
+.. note::
+
+  On DKRZ/Levante, a user can't have more than 20 SLURM jobs running at a time.
+  As soon as a job is finished, the next one should start
+
+Submit the batch scripts that will run all recipes. Assemble some statistics so that issues with certain recipes
+can be followed-up, and document this information in the release issue, examples:
+
+- number of successfully run recipes
+- number of failed recipes with Diagnostic error (can they be fixed? Can the fixes be included in the release?)
+- number of recipes that are missing data
+- number of recipes that have various other issues (and document them)
+
+To parse the output of all these runs use the `parse_recipes_output.py` Python script, included at the
+same locations where the generation script is.
+
+Running the comparison
+----------------------
+
+To compare the newly produced output from running all recipes, follow these steps below.
+
+Login and access to the DKRZ esmvaltool virtual machine (VM) - results from recipe runs
+are stored on the VM; login with:
+
+.. code-block:: bash
+
+  ssh user@esmvaltool.dkrz.de
+
+where `user` is your DKRZ/Levante user name; then get and install miniconda on the VM, and
+if you have a Miniconda installer already downloaded in your Levante $HOME
+
+.. code-block:: bash
+
+  scp Miniconda3-py39_4.12.0-Linux-x86_64.sh user@esmvaltool.dkrz.de:~
```

Comment:
Please mention that conda environments should not be created in the home directory because it is on a very small disk, but rather in a directory with your username under `/mnt/esmvaltool_disk2/work/`

Reply:
very good point indeed! Juast added a warning about that